test/odbc/test_pg_datatype_float.cpp

```diff
@@ -0,0 +1,610 @@
+#include <gtest/gtest.h>
```

Comment:
Additional tests to add
- Comparison Operators (<, <=, etc)
- Comparison Functions (Min, Max, etc)
- Arithmetic Operators (+, -, etc)
- Arithmetic Functions (pow, log, etc)
- Unique and Composite Key